_1327/polls/views.py

```diff
@@ -17,22 +17,32 @@
 from _1327.main.utils import abbreviation_explanation_markdown
 from _1327.polls.models import Poll
 from _1327.user_management.shortcuts import check_permissions
+from guardian.core import ObjectPermissionChecker
 
 
 def index(request):
 	running_polls = []
 	finished_polls = []
 	upcoming_polls = []
+
+	polls = Poll.objects.all().order_by('-end_date').prefetch_related('participants')
+
+	# Prefetch user permissions
+	user_checker = ObjectPermissionChecker(request.user)
```

Comment:
the problem here is that you only check for permissions the user has, but we need to check the group permissions and ip range group permissions as well (see `minutes/views.py:20`).
before we change the permission checks in too many places (before we only had `has_perm` which is overwritten to fulfill our needs) we need to come up with a general solution. maybe you could discuss that with @Bartzi.